src/libs/NumberFormatUtils.ts

```diff
@@ -1,12 +1,19 @@
+import moize from 'moize';
 import type {ValueOf} from 'type-fest';
 import type CONST from '@src/CONST';
 
+const numberFormatter = moize(Intl.NumberFormat, {
+    isDeepEqual: true,
+    maxSize: 10,
+    profileName: 'Intl.NumberFormat',
+});
+
 function format(locale: ValueOf<typeof CONST.LOCALES>, number: number, options?: Intl.NumberFormatOptions): string {
```

Comment:
What's the difference between memoizing the `format` vs `Intl.format` calls?

Reply:
Sorry, I am not sure if I get the question right - could you please tell me more details? Thanks!

We are caching just `Intl.NumberFormat`. Both `format` and `formatToParts` are the consumers of the cached class.

Reply:
Yeah I got that. I meant, instead of caching `Intl`, what if we cache `format` and `formatToParts`. Does that give us any benefit?

```
const formatLocal = moize(format, {
    isDeepEqual: true,
    maxSize: 10,
    profileName: 'format',
});

const formatToPartsLocal = moize(formatToParts, {
    isDeepEqual: true,
    maxSize: 10,
    profileName: 'formatToParts',
});



export { formatLocal as format, formatToPartsLocal as formatToParts }

```

Reply:
I'd say it won't help as they rely on same API (the `Intl.NumberFormat`) so caching the root is more straightforward and thus probably better performance-wise (although I did not measure it)